src/python/qeqiskit/noise/basic_test.py

```diff
@@ -34,3 +40,31 @@ def test_get_qiskit_noise_model_no_device(self):
                 QiskitBackendNotFoundError,
                 lambda: get_qiskit_noise_model(device, api_token=self.ibmq_api_token),
             )
+    
+    def test_t_1_t_2_noise_models(self):
+        for noise in self.t_1_t_2_models:
+            self.assertIsInstance(noise(self.T_1, self.T_2, self.t_step), AerNoise.NoiseModel)
+
+    def test_amplitude_damping_model(self):
+        self.assertIsInstance(create_amplitude_damping_noise(self.T_1, self.t_step), AerNoise.NoiseModel)
+
+    def test_phase_damping_noise(self):
+        self.assertIsInstance(create_phase_damping_noise(self.T_2, self.t_step), AerNoise.NoiseModel)
```

Comment:
@akataba Is there something more we can be testing here? Like can we expect the noise model to have a certain property depending on `self.T_2` and `self.t_step`?

Reply:
If so, I'd be inclined to add this now if it's relatively simple. Or just make sure to have a new PR relatively soon with this change

Reply:
I can't think of any  property that has any dependence apart `self.T_2,`` self.T_1` or `self.t_step` simply being input arguments which then are used as inputs into the kraus operators.